python/dedup_units.py

```diff
@@ -42,6 +42,10 @@ def resolve_all_projects(df):
     resolved = df.groupby(["project_id"], as_index=False).apply(resolve_project)
     resolved = resolved[["project_id", "source", "units_gross", "units_net", "record_id"]]
     print(resolved.head())
+
+    # removing duplicate record after 
```

Comment:
I think the comment here should be focused on why we want to remove duplicate records - anytime we can write out the logic in layman's terms it will help the engineer figure out what's going on

Reply:
excellent question. Almost forgot all about this after a year. Trying to remember and will write better descriptions

Reply:
this turns out to be a more significant issue. I think this thing I did might not be legal and I need to look more upstream for a better solution. Stay tuned for this @mbh329